auth-api/src/auth_api/resources/v1/permissions.py

```diff
@@ -33,8 +35,15 @@
 def get_membership_permissions(org_status, membership_type):
     """Get a list of all permissions for the membership."""
     try:
-        case = request.args.get("case")
-        permissions = PermissionsService.get_permissions_for_membership(org_status.upper(), membership_type.upper())
+        case: str = request.args.get("case")
+        include_all_permissions: bool = string_to_bool(request.args.get("includeAllPermissions", "False"))
```

Comment:
backwards compatible and to allow for flexibility for the UI where sometimes we want membership to accounts permissions and not including staff / external staff permissions